bedrock/mozorg/templates/mozorg/home/home-a.html

```diff
@@ -0,0 +1,280 @@
+      </div>
+    </div>
+
+    <div id="aframe-wrapper"></div>
+
+    <div class="section-body">
+      <div class="content">
+        <h3>{{ _('Introducing A-Frame') }}</h3>
+        <p>{{ _('Check out this ground-breaking, open-source framework for easily creating web-based virtual reality experiences.') }}</p>
+        <p class="section-cta"><a href="#">{{ _('Learn more') }}</a></p>
+      </div>
+    </div>
+  </section>
+
+  <section class="section section-firefox" id="firefox">
+    <div class="event-horizon">
+      <div class="section-lead">
+        <div class="content">
+          <h2>{{ _('Browse freely') }}</h2>
+        </div>
+      </div>
+    </div>
+    <div class="mission-control">
+      <div class="sky-line"></div>
+      <div class="launch-area">
+        <div class="rocket"></div>
+      </div>
+      <div class="crater"></div>
+      <div class="section-body">
+        <div class="content">
+          <p>{{ _('With Firefox for desktop, iOS and Android, you can have a fast, reliable web experience everywhere you go. To sync your browser across all your devices, sign up for a Firefox Account today.') }}</p>
+          <p class="section-cta"><a href="{{ url('firefox.family.index') }}">{{ _('Discover more') }}</a></p>
+        </div>
+      </div>
+    </div>{#--/.mission-control --#}
+  </section>
+
+  <section class="section section-externals" id="externals">
+    <ul class="content">
+      <li>
+        <a class="add-ons" rel="external" href="https://addons.mozilla.org/">
+          <h3>{{ _('Add-ons') }}</h3>
+          <p>{{ _('Compare prices, check the weather, listen to music, send a tweet and more right from Firefox.') }}</p>
+        </a>
+      </li>
+      <li>
+        <a class="careers" rel="external" href="https://careers.mozilla.org/">
+          <h3>{{ _('Careers') }}</h3>
+          <p>{{ _('Learn about the benefits of working at Mozilla and view open positions around the world.') }}</p>
+        </a>
+      </li>
+      <li>
+        <a class="help" rel="external" href="https://support.mozilla.org/">
+          <h3>{{ _('Need help?') }}</h3>
+          <p>{{ _('Get answers to your questions about Firefox and all Mozilla products from our support team.') }}</p>
+        </a>
+      </li>
+    </ul>
+  </section>
+
+  <aside class="section section-newsletter" id="newsletter-subscribe">
+    <div class="content">
+    {% if LANG.startswith('en-') %}
+      {{ email_newsletter_form(newsletters='mozilla-foundation', title=_('Love the Web?'), subtitle=_('Get the Mozilla newsletter and help us keep it open and free.'), button_class='light') }}
+    {% else %}
+      {{ email_newsletter_form(button_class='light') }}
+    {% endif %}
+    </div>
+  </aside>
+
+</main>
+{% endblock %}
+
+{% block site_footer %}
+<footer class="section section-footer" id="footer">
```

Comment:
Yeah, it's possible Concept B will have a different footer so let's leave it for now. I also copied the whole thing instead of using the include we already have simply because I wanted a slightly different structure. We'll rethink all this when we work on consolidating a new base template.